src/styles/main.scss

```diff
@@ -0,0 +1,129 @@
+$bodyfont: Roboto, sans-serif;
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0 auto;
```

Comment:
The `body` tag should not have `margin: 0 auto;`. This rule is used for centering blocks, but the body is not a block that should be centered in this way.